### ISSUE TITLE
Fix wallet ID usage in RPC when created from CLI

### ISSUE THREAD

**Issue by ghost** [nanocurrency/raiblocks#42]
The wallet ID is cached in memory on the node and doesn't get correctly updated

**Comment by rkeene** [comment]
Is this still a problem ?

**Comment by argakiig** [comment]
Yes, as far as I know this has to do with the fact thar if you have a node running and call another instance of the node to create a wallet or account via rpc it is calling an inactive node rather than the currently running process, so different namespaces

**Comment by renesq** [comment]
Any wallet-altering CLI action requires the node (daemon) to be restarted afterwards in order to have them take effect and be accessible from CLI.

I also noticed CLI-created tipbot wallets to behave pretty strange in general, so we deleted those and created them with RPC instead. I don't remember if we had the daemon running while doing it, but I normally tend to avoid this because I presume that the database could potentially break when multiple instances  write to it. 

Also remember that the pending "wallet_separation" branch #843 may be related to the improvement of the behavior.
